images/scripts/rhel.setup

```diff
@@ -184,6 +184,7 @@ sssd-dbus \
 "
 
 TEST_PACKAGES="\
+acl \
```

Comment:
Do we know for sure that's on all RHEL?

Reply:
Yes, all the way back to RHEL 3. It just used to be part of the base system (systemd dependency) until 9, so we never installed it explicitly.